src/main/java/tech/jhipster/lite/module/domain/JHipsterModule.java

```diff
@@ -277,8 +279,16 @@ public JHipsterModuleBuilder documentation(DocumentationTitle title, JHipsterSou
       String target = "documentation/" + title.filename() + source.extension();
       files().add(source, to(target));
 
-      String markdownLink = "- [" + title.get() + "](" + target + ") " + LINE_BREAK + JHIPSTER_DOCUMENTATION_NEEDLE;
-      optionalReplacements().in("README.md").add(text(JHIPSTER_DOCUMENTATION_NEEDLE), markdownLink);
+      String markdownLink = "- [" + title.get() + "](" + target + ") ";
```

Comment:
do we need this additional space at the end ?

```suggestion
      String markdownLink = "- [" + title.get() + "](" + target + ")";
```

Reply:
I don't think so :)